src/pattern_library/components/App.vue

```diff
@@ -0,0 +1,396 @@
+						<q>We think we understand the rules when we become adults but what we really experience is a narrowing of the imagination.</q>
+					</blockquote>
+
+					<blockquote class="donation-comment">
+						<p class="repel donation-comment__meta"><strong>€15.00 from David Lynch</strong><em>24.01.2025 um 01:18 Uhr</em></p>
+						<q>We think we understand the rules when we become adults but what we really experience is a <em>narrowing of the imagination</em>.</q>
+					</blockquote>
+
+					<nav class="pagination">
+						<ul class="cluster">
+							<li><button disabled><span class="visually-hidden">Previous Page</span><ChevronLeftIcon/></button></li>
+							<li><button aria-current="true"><span class="visually-hidden">Page </span>1</button></li>
+							<li><button><span class="visually-hidden">Page </span>2</button></li>
+							<li><button><span class="visually-hidden">Page </span>3</button></li>
+							<li><button><span class="visually-hidden">Page </span>4</button></li>
+							<li><button><span class="visually-hidden">Page </span>5</button></li>
+							<li><button><span class="visually-hidden">Page </span>6</button></li>
+							<li><button><span class="visually-hidden">Page </span>7</button></li>
+							<li><button><span class="visually-hidden">Page </span>8</button></li>
+							<li><button><span class="visually-hidden">Page </span>9</button></li>
+							<li><button><span class="visually-hidden">Page </span>10</button></li>
+							<li><button><span class="visually-hidden">Next Page</span><ChevronRightIcon/></button></li>
+						</ul>
+					</nav>
+				</div>
+
+				<div class="content-card flow">
+					<div class="section-heading">
+						<h2>More Information Toggle</h2>
+						<hr/>
+					</div>
+
+					<button class="button" data-button-style="hollow">Enter Postal Address</button>
+
+					<details class="more-info-toggle">
+						<summary><small>More Information</small></summary>
+						<p><small>This is all the more info. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloremque laudantium nisi possimus, quas repudiandae veritatis? Architecto blanditiis, et impedit in incidunt inventore laborum molestiae molestias odio pariatur rem sit veniam.</small></p>
+					</details>
+				</div>
+
+				<div class="content-card flow">
+					<div class="section-heading">
+						<h2>Modal</h2>
+						<hr/>
+					</div>
+
+					<button class="button" @click="modalIsVisible = true">Show Modal</button>
+				</div>
+			</main>
+
+			<aside>
+				<div class="sticky flow">
+					<div class="content-card flow" data-sidebar-card>
+						<p>Sidebar content</p>
+					</div>
+					<div class="content-card flow" data-sidebar-card>
+						<p>Sidebar content</p>
+					</div>
+					<div class="content-card flow" data-sidebar-card>
+						<p>Sidebar content</p>
+					</div>
+				</div>
+			</aside>
+		</div>
+
+		<div class="grid" data-layout="50-50">
```

Comment:
I thought I understood the grid, but this example on large view ports makes me doubt my understanding. As far as I understand, this should create a two-column grid. But why does the grid not distribute the 2nd and 3rd element as 50% columns and instead makes them small items that don't go well with the overall symmetry, giving an almost-but-not-quite impression of three columns in the 2nd row? Or is that a bug?